package.json

```diff
@@ -23,8 +23,8 @@
   },
   "lint-staged": {
     "*.js": [
-      "yarn test",
-      "prettier --single-quote --no-semi --write",
+      "yarn test && :",
```

Comment:
This was added to make sure the provided arguments from `lint-staged` don't cause any trouble.

See [this comment](https://github.com/okonet/lint-staged/issues/174#issuecomment-437468837) from the `lint-staged` repo.